modules/run-consul/run-consul

```diff
@@ -329,6 +373,32 @@ function run {
         datacenter="$2"
         shift
         ;;
+      --enable-gossip-encryption)
+        enable_gossip_encryption="true"
```

Comment:
`enable_gossip_encryption` and all the other vars should be declared above as `local` vars with reasonable defaults (`false` for the bools, empty for the strings).

Reply:
Will do.